docs/01-app/01-getting-started/07-caching-and-revalidating.mdx

```diff
@@ -0,0 +1,299 @@
+---
+title: How to cache and revalidate components and functions
+nav_title: Caching and Revalidating
+description: Learn how to cache and revalidate components and functions in your Next.js application.
+related:
+  title: API Reference
+  description: Learn more about the features mentioned in this page by reading the API Reference.
+  links:
+    - app/api-reference/config/next-config-js/dynamicIO
+    - app/api-reference/directives/use-cache
+    - app/api-reference/functions/cacheLife
+    - app/api-reference/functions/cacheTag
+    - app/api-reference/functions/revalidateTag
+    - app/api-reference/functions/revalidatePath
+---
+
+> **Warning:** The content below assumes the [`dynamicIO` config option](/docs/app/api-reference/config/next-config-js/dynamicIO) is enabled in your application. This option was introduced in Next.js 15 canary.
+
+## Caching
+
+In Next.js, you can cache the output of a [component](#components) or [function](#functions) to reduce the need to re-execute work for every user request.
+
+Caching is useful for data that doesn't change often or is shared across multiple users, as you can **prerender** at build time, and reuse the static content for all requests.
```

Comment:
I wonder if we should talk about prerendering this high up, because you could also cache this at runtime (not prerendered) right?

Reply:
Update the wording slightly to remove "prerendering" and avoid confusion. I think the runtime / in-memory cache might be too detailed for this section. but we should ensure it's documented in the `use cache` page.